MyCryptoHQ/MyCrypto
# Add WebUSB support for Ledger devices

### Description

This PR adds [WebUSB](https://wicg.github.io/webusb/) support using Ledger's `@ledgerhq/hw-transport-webusb`. This solves issues like the [U2F timeout](https://support.ledger.com/hc/en-us/articles/360018810413-U2F-timeout-in-Chrome-browser) caused by a recent Chromium update. Currently WebUSB is only supported by Chrome and Opera. The "old" U2F method is used in browsers that don't support WebUSB. This only applies to the web version of MyCrypto, the desktop version is unchanged.

I also updated other Ledger libraries, to make sure we don't have any duplicates.

### Changes

* Use `@ledgerhq/hw-transport-webusb` as transport method if it's supported.

### Steps to Test

1. Access a Ledger device with MyCrypto as usual.
2. Select your device (see screenshot below).
3. Try sending a transaction, signing a message, etc.

### Screenshots

![](https://i.imgur.com/3eSvjbE.png)
